frontend/src/v5/ui/controls/search/searchInput/searchInput.component.tsx

```diff
@@ -15,61 +15,76 @@
 
 type ISearchInput = {
-	/**
-	 * Callback when the clear button is clicked.
-	 * Note: the clear button only appears when the controls is controlled (read more https://reactjs.org/docs/forms.html#controlled-components)
-	 */
-	onClear?: () => void;
-} & TextFieldProps;
+	variant?: 'filled' | 'outlined',
+	placeholder?: string,
+} & Partial<AutocompleteProps<TextFieldProps, boolean, undefined, undefined, typeof SearchChip>>;
 
-export const SearchInput = ({ onClear, value, variant = 'filled', ...props }: ISearchInput): JSX.Element => {
-	const { query, setQuery } = useContext(SearchContext);
+export const SearchInput = ({ variant = 'filled', multiple, placeholder, ...props }: ISearchInput): JSX.Element => {
+	const { queries, setQueries } = useContext(SearchContext);
 
-	const onChange = (event: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
-		setQuery(event.currentTarget.value);
-		if (props.onChange) {
-			props.onChange(event);
-		}
+	const onChange = (event: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>, newValue) => {
+		// This is for when enter is pressed and the chips change
+		if (!multiple) return;
+		setQueries(newValue);
 	};
 
-	const onClickClear = () => {
-		if (onClear) {
-			onClear();
-		}
-
-		setQuery('');
+	const onInputChange = (event: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>, newValue) => {
+		// This is for when the textfield value changes by typing
+		if (multiple) return;
+		setQueries([newValue]);
 	};
 
-	const val = query || value || '';
+	const removeQuery = (index) => setQueries((prev) => prev.filter((el, i) => index !== i));
 
 	return (
-		<TextField
-			value={val}
-			InputProps={{
-				startAdornment: (
-					<StartAdornment>
-						<SearchIcon />
-					</StartAdornment>
-				),
-				endAdornment: (
-					<EndAdornment $isVisible={Boolean(val)}>
-						<IconButton onClick={onClickClear} size="large">
-							<CloseIcon />
-						</IconButton>
-					</EndAdornment>
-				),
-			}}
-			variant={variant}
-			{...props}
+		<Autocomplete
+			value={queries}
+			open={false}
+			freeSolo
+			multiple={multiple}
+			options={[]}
+			clearIcon={<CloseIcon />}
+			getOptionLabel={(labelObject) => labelObject[0] ?? ''} // This silences a console error
+			renderTags={(
+				values: any[],
+			) => values.map((query: any, index: number) => (
+				<SearchChip
+					key={query}
+					label={query}
+					onDelete={() => removeQuery(index)}
+				/>
+			))}
 			onChange={onChange}
+			onInputChange={onInputChange}
+			{...props}
+			renderInput={(params: any) => (
+				<TextField
+					variant={variant}
+					multiline={multiple}
+					placeholder={placeholder || formatMessage({ id: 'searchInput.defaultPlaceholder', defaultMessage: 'Search...' })}
+					{...params}
+					InputProps={{
+						...params.InputProps,
+						startAdornment: (
+							<>
+								<StartAdornment>
+									<SearchIcon />
+								</StartAdornment>
+								{params?.InputProps?.startAdornment}
```

Comment:
Is this actually been used anywhere? and if so, should it not replace the startAdornment?

Reply:
Yes, so autocomplete adds the values as chips when `multiple=true`. These are added as startAdornments which means if I were to put the searchIcon as the startAdornment like normal it will override any chips